docs/integration.md

```diff
@@ -0,0 +1,16 @@
+# Identity Directory and Blockchain Node Integration
+
+The **blockchain** node is provided to the Identity Directory as a Docker image, also
```

Comment:
It isn't connected automatically, so that's just an option. Maybe we could change that paragraph to something like "Identity Directory client app has an option to chose to which **blockchain node** it will connect. There are predefined options like
Polkadot and Kusama offical websockets or a custom one where the user will type in the address of a node. There are also private ones that are running on AWS EC2 instances."